modules/svg/image_loader_svg.cpp

```diff
@@ -82,6 +82,13 @@ Error ImageLoaderSVG::create_image_from_utf8_buffer(Ref<Image> p_image, const Pa
 	uint32_t width = round(fw * p_scale);
 	uint32_t height = round(fh * p_scale);
 
+	// Twg could return in some web-based cases,
+	// or the size could be scaled and rounded to 0. Render 1x1 px.
+	if (width < 1 || height < 1) {
+		width = 1;
+		height = 1;
+	}
```

Comment:
I guess you meant "tvg" instead of "twg"?

This would force e.g. `(1024, 0.5)` to `(1, 1)` instead of an IMO more expected `(1024, 1)`.

I'd suggest instead:
```suggestion
	uint32_t width = MAX(1, round(fw * p_scale));
	uint32_t height = MAX(1, round(fh * p_scale));
```

Reply:
Yes, a line could be better. Godot isn't just an image viewer:) For example, the image might be downscaled to a line and used for some blur effects.( It should be documented by an eng. native speaker as well
https://docs.godotengine.org/en/latest/tutorials/assets_pipeline/importing_images.html )
![image](https://user-images.githubusercontent.com/4047289/230136725-aeeb3089-b30d-4d65-85f1-cc532980d107.png)